gnovm/tests/imports.go

```diff
@@ -567,6 +567,53 @@ func testPackageInjector(store gno.Store, pn *gno.PackageNode) {
 				m.Context = ctx
 			},
 		)
+		pn.DefineNative("TestSetPrevRealm",
+			gno.Flds( // params
+				"", gno.AnyT(),
+			),
+			gno.Flds( // results
+			),
+			func(m *gno.Machine) {
+				arg0 := m.LastBlock().GetParams1().TV
+
+				switch arg0.T.String() {
+				case "std.Address":
+					// Set PrevRealm as an user
+					addr := arg0.GetString()
+					ctx := m.Context.(stdlibs.ExecContext)
+					ctx.OrigCaller = crypto.Bech32Address(addr)
+					m.Context = ctx
+
+				case "string":
+					// Set PrevRealm as a realm
+					realm := arg0.GetString()
+
+					// Set PkgPath
+					for i := m.NumFrames() - 1; i > 0; i-- {
+						fr := m.Frames[i]
+						if fr.LastPackage == nil || !fr.LastPackage.IsRealm() {
+							if i != 1 {
+								fr.LastPackage = &gno.PackageValue{
+									PkgPath: "gno.land/r/this_is_dummy_pkg_path",
```

Comment:
Do we want this kind of non-standard package name, that is defined here? cc @moul

Reply:
I wanted pkg_path value used on that line should never exist on chain. If it exists maybe there will be some side effect ( or maybe I'm worrying too much :D )